### PR TITLE
Add substrata Xresources colors

### PR DESCRIPTION
**Pull request by just1602**

### CODE REVIEW

File: xresources/substrata.xresources [hunk 1]
```diff
@@ -0,0 +1,20 @@
+! Substrata theme
```
Comment (lunacookies) [inline xresources/substrata.xresources]:
```suggestion
! Substrata Theme <https://github.com/arzg/vim-substrata>
```

File: xresources/substrata.xresources [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+*.color3: #ab924c
+*.color11: #d2b45f
+*.color4: #8296b0
+*.color12: #a0b9d8
+*.color5: #a18daf
+*.color13: #c6aed7
+*.color6: #659ea2
+*.color14: 7dc2c7
+*.color7: #b5b4c9
+*.color15: #f8ecfe
```
Comment (lunacookies) [inline xresources/substrata.xresources]:
```suggestion
*.color15: #f0ecfe
```

File: xresources/README.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# Xresources substrata theme
+
+If you want to use this theme, you can simply copy the content of `substrata.xresources` in your `~/.Xresources`. Otherwise, you can include it in your `~/.Xresources` by adding the following line to your configuration :
+
+`#include "/path/to/substrata.xresources"`
```
Comment (lunacookies) [inline xresources/README.md]: Would it be ok if you removed this README, instead moving the text from here into the root project’s README?

**Review by lunacookies — CHANGES_REQUESTED** [review]

File: xresources/README.md (hunk 3 above)
Comment (just1602) [reply xresources/README.md]: Nope, no stress at all, I'll fix this tonight ! :)

File: README.md [hunk 4]
```diff
@@ -30,6 +30,12 @@ If you’re using a GUI, then Substrata should work out of the box. However, if
 
 A preset is included for iTerm.
 
+## Xresources substrata theme
+
+If you want to use this theme, you can simply copy the content of `xresources/substrata.xresources` in your `~/.Xresources`. Otherwise, you can include it in your `~/.Xresources` by adding the following line to your configuration :
+
+`#include "/path/to/substrata.xresources"`
```
Comment (lunacookies) [inline README.md]:
```suggestion
    #include "/path/to/substrata.xresources"
```

File: README.md [hunk 5]
```diff
@@ -30,6 +30,12 @@ If you’re using a GUI, then Substrata should work out of the box. However, if
 
 A preset is included for iTerm.
 
+## Xresources substrata theme
+
+If you want to use this theme, you can simply copy the content of `xresources/substrata.xresources` in your `~/.Xresources`. Otherwise, you can include it in your `~/.Xresources` by adding the following line to your configuration :
```
Comment (lunacookies) [inline README.md]:
```suggestion
If you want to use this theme, you can simply copy the contents of `xresources/substrata.xresources` into your `~/.Xresources`. If you would like to keep the theme in a separate file, however, you could instead include it in your `~/.Xresources` by adding the following line:
```

**Review by lunacookies — CHANGES_REQUESTED** [review]

**Pull request merged**